intel/custom_resource.py

```diff
@@ -106,17 +110,9 @@ def save(self):
 
     def exists(self, namespace="default"):
         """Check if custom resource definition exists"""
-        resource_path = "/".join([
-            "/apis",
-            self.group,
-            "v1",
-            "namespaces", namespace,
-            self.plural_name
-        ])
-
```

Comment:
Now I thought it is path to custom objects not to custom resource definition. So I've changed path